shared/js/sanitizer.js

```diff
@@ -74,8 +74,11 @@
      * Unwrap safe HTML created by createSafeHTML or a custom replacement that
      * underwent security review.
      */
-    unwrapSafeHTML: function (htmlObject) {
-      return htmlObject.__html;
+    unwrapSafeHTML: function (...htmlObjects) {
+      var markupList= htmlObjects.map(function(obj) {
```

Comment:
Sorry for the delay. Bug 1209416 is ready to be checked in now, just waiting for a sherrif to land it.